### PR TITLE
Add part regarding the soccer ipm

### PR DESCRIPTION
**Pull request by Flova**

### CODE REVIEW

File: soccer.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
+.. _SoccerIPM:
+
+Soccer IPM
+##########
+
+Overview
+========
+
+Soccer IPM is a ROS package that provides a node to project 2D image space detections of soccer related objects (e.g., balls, goal posts, robots, etc.) to 3D world space.
```
Comment (jaagut) [inline soccer.rst]:
```suggestion
Soccer IPM is a ROS package that provides a node to project 2D image space detections of soccer related objects (e.g., balls, goal posts, robots, etc.) to 3D relative space. 
```

File: soccer.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+How do I use it?
+================
+
+First of all, you need to download the repo as well as its dependencies and build the package in your workspace.
+You can do this by running the following commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/ros-sports/soccer_ipm.git src/soccer_ipm
+   rosdep install --from-paths src --ignore-src -r -y
```
Comment (jaagut) [inline soccer.rst:51]: rosdep might need a little bit more setup. We could probably just reference a guide for this

File: soccer.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+You probably want to do this using a launch file. For example, you can use the following launch and config files:
+
+.. code-block:: xml
+
+   <node pkg="soccer_ipm" exec="ipm" name="soccer_ipm" output="screen">
+      <param from="$(find-pkg-share your_package)/config/soccer_ipm.yaml" />
+   </node>
+
+.. code-block:: yaml
```
Comment (jaagut) [inline soccer.rst:76]: This code block does not render in the GitHub preview for some reason

Comment (Flova) [reply soccer.rst:76]: Sphinx renders it so it is fine i guess

File: soccer.rst (hunk 2 above)
Comment (Flova) [reply soccer.rst:51]: We don't do this in the other guides

**Review by jaagut — APPROVED** [review]

**Pull request merged**